extism_cli/__init__.py

```diff
@@ -159,6 +159,12 @@ def asset_prefix(self, libc: str = "gnu"):
                   action="store_true",
                   help="Print link flags")
 
+# Call subcommand
+call = subparsers.add_parser("call")
+call.add_argument("wasm", help='WASM file')
+call.add_argument("--input", default=None, help='Plugin input')
+call.add_argument("function", help='Function name')
```

Comment:
Could maybe use some flags to set the debug level? or is that set another way?

Reply:
You read my mind! I just pushed a commit to do this